src/components/Popover/_helpers/cleanPlacementStyle.js

```diff
@@ -0,0 +1,29 @@
+export default (placementStyle) => {
+  const validProps = [
+    'position',
+    'inset',
+    'inset-inline-start',
+    'inset-inline-end',
+    'inset-block-start',
+    'inset-block-end',
+    'top',
+    'right',
+    'bottom',
+    'left',
+    'translate',
+    'transform-origin',
+  ];
+
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
In dev mode the allowed styles are already handled by propType validation. I dont think we need this explicit check at all.

If we do, the errors in console caused by `bject.keys(placementStyle)` when `placementStyle === null` need to be fixed,